troubleshooting.md

```diff
@@ -47,6 +47,14 @@ will disable SELinux separation for the container.
 
 $ podman run --security-opt label=disable -v ~:/home/user fedora touch /home/user/file
 
+In cases where the container image runs as a specific, non-root user, though, the
+solution is to fix the user namespace.  This would include container images such as
+the Jupyter Notebook image (which runs as "jovyan") and the Postgres image (which runs
+as "postgres").  In either case, use the `--userns` switch to map user namespaces,
+most of the time by using keep_id option.
+
+$ podman run -v "$PWD":/home/jovyan/work --userns=keep_id jupyter/scipy-notebook
+
 ---
```

Comment:
Do we need to also call `system migrate` as we show in #10 in this doc?

Reply:
No migrate is only needed if you have to modify /etc/subuid and /etc/subgid.

Reply:
I was asking as I assumed "fix the usernamespace" meant jiggling the settings in /etc/subuid and /etc/subgid.